procedures/unit-testing-tracing.ipf

```diff
@@ -210,6 +225,10 @@ static Function SetupTraceProcedures(string procWinList, string traceOptions)
 		procWin = StringFromList(i, procWinList)
 		[procText, funcPath, markLines] = AddTraceFunctions(procWin, i)
 		if(!UTF_Utils#IsEmpty(funcPath))
+			gridIndex = UTF_Utils_Vector#AddRow(procTextGrid)
```

Comment:
I think this breaks it, because originally procWinList is saved, here you save only a subset.